tests/win_sec_attr.c

```diff
@@ -50,27 +50,27 @@ int main ()
     PACL pACL;
     DWORD ret2;
     int ret3;
-    void * void_ret_value = NULL;
-    size_t void_ret_size = sizeof(void_ret_value);
+    void *void_ret_value = NULL;
+    size_t void_ret_size = sizeof (void_ret_value);
```

Comment:
do we need a scratch variable here?  would sizeof (void *) work equally well without the void_ret_value declaration?